Source/IGListBindingSectionController.h

```diff
@@ -77,6 +77,8 @@ NS_SWIFT_NAME(ListBindingSectionController)
  */
 - (void)updateAnimated:(BOOL)animated completion:(nullable void (^)(BOOL updated))completion;
 
+- (void)moveObjectFromIndex:(NSInteger)sourceIndex toIndex:(NSInteger)destinationIndex NS_REQUIRES_SUPER;
```

Comment:
Can you add some doc headers? I think copy/paste the superclass ones but add a note about why super is required?

Reply:
Sure, I committed the changes.